pkg/services/folder/folderimpl/folder_test.go

```diff
@@ -37,19 +36,6 @@ import (
 var orgID = int64(1)
 var usr = &user.SignedInUser{UserID: 1, OrgID: orgID}
 
-func TestIntegrationProvideFolderService(t *testing.T) {
```

Comment:
removed this test as it is testing Accesscontrol registering Scoperesolvers

Reply:
i understand that it is part of the folder service, on how many scoped resolvers it should get

Reply:
can add it back if we do not want to remove it

Reply:
I think we should leave this test in. It checks if the RBAC scope resolvers are correctly registered, so it doesn't hurt to have it.